stubs/pexpect/pexpect/_async.pyi

```diff
@@ -0,0 +1,19 @@
+import asyncio
+from _typeshed import Incomplete
+
+@asyncio.coroutine
+def expect_async(expecter, timeout: Incomplete | None = None): ...
+@asyncio.coroutine
+def repl_run_command_async(repl, cmdlines, timeout: int = -1): ...
```

Comment:
Using `@coroutine` doesn't work anymore:

```suggestion
async def expect_async(expecter, timeout: Incomplete | None = None): ...
async def repl_run_command_async(repl, cmdlines, timeout: int = -1): ...
```

This should also get rid of the mypy and pyright failures.